zip-frost.rst

```diff
@@ -0,0 +1,441 @@
+signatures in the Zcash protocol, for the Sapling and Orchard network upgrades.
+
+
+Motivation
+==========
+
+In the Zcash protocol, Spend Authorization Signatures are employed to authorize
+a transaction. The ability to generate these signatures with the user's
+private key is what effectively allows the user to spend funds.
+
+This is a security-critical step, since anyone who obtains access to the private
+key will be able to spend the user's funds. For this reason, one interesting
+possibility is to require multiple parties to allow the transaction to go through.
+This can be accomplished with threshold signatures, where the private key is
+split between parties in a way that a threshold (e.g. 2 out of 3) of them must
+sign the transaction in order to create the final signature. This enables scenarios
+such as users and exchanges sharing custody of a wallet, for example.
+
+FROST is one of such threshold signature protocols. However, it can't be used as-is
+since the Zcash protocol also requires re-randomizing public and private keys
+to ensure unlinkability between transactions. This ZIP specifies a variant of
+FROST with re-randomization support.
+
+
+Requirements
+============
+
+- All signatures generated by following this ZIP must be verified successfully.
+  as Sapling or Orchard spend authorization signatures.
+- The signatures generated by following this ZIP should meet the security criteria
+  for Signature with Re-Randomizable Keys as specified in the Zcash protocol [#protocol-concretereddsa]_.
+- The threat model described below must be taken into account.
+
+Threat Model
+------------
+
+In normal usage, a Zcash user follows multiple steps in order to generate a
+shielded transaction:
+
+- The transaction is created.
+- The transaction is signed with a re-randomized version of the user's spend
+  authorization private key.
+- The zero-knowledge proof for the transaction is created with the randomizer
+  as an auxiliary (secret) input, among others.
+
+When employing re-randomized FROST as specified in this ZIP, the goal is to
+split the spend authorization private key :math:`\mathsf{ask}` among multiple
+possible signers. This means that the proof generation will still be generated
+by a single participant, likely the one that created the transaction in the first
+place. Note that this user already controls the privacy of the transaction since
+they are responsible for creating the proof.
+
+This fits well into the "Coordinator" role from the FROST specification [#frost-protocol]_.
+The Coordinator is responsible for sending the message to be signed to all participants,
+and to aggregate the signature shares.
+
+With those considerations in mind, the threat model considered in this ZIP is:
+
+- The Coordinator is trusted with the privacy of the transaction (which includes
+  the unlinkability property). A rogue Coordinator will be able to break
+  unlinkability and privacy, but should not be able to create signed transactions
```

Comment:
Can I re-iterate all participants are able to break the unlinkability of the TX, not just the Coordinator? The discrete log of the re-randomization is sufficient as a proof.

Reply:
This item is only about the Coordinator, the next item talks about key share holders. But I made it more explicit in 1342ead

Reply:
Ah, sorry, missed that. My bad there. Thanks for covering ti :+1: